Behavioral/State/ContextOrder.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace DesignPatterns\Behavioral\State;
+
+class ContextOrder extends StateOrder
+{
+    public function getState():StateOrder
```

Comment:
PHP 7 sintax should go to `php7` branch (https://github.com/domnikl/DesignPatternsPHP/pull/169#issuecomment-248949061). Master branch still PHP 5.6 compatible